src/core/sys/freebsd/sys/event.d

```diff
@@ -43,15 +43,33 @@ extern(D) void EV_SET(kevent_t* kevp, typeof(kevent_t.tupleof) args)
     *kevp = kevent_t(args);
 }
 
-struct kevent_t
+version(FreeBSD12)
```

Comment:
Let's go with `static if (__FreeBSD_version >= INO64_FIRST)` to avoid this becoming a maintenance task for future FBSD releases. Also see https://github.com/dlang/dmd/pull/8567/files#r211100860.